examples/test/PivotTableDynamic_test.js

```diff
@@ -197,3 +204,42 @@ test('should display sticky header only for grouped attributes', async (t) => {
     await checkNodeIsTransparent(t, Selector(`${PINNED_TOP_ROW} ${CELL_0_3}`), true);
     await checkNodeIsTransparent(t, Selector(`${PINNED_TOP_ROW} ${CELL_0_4}`), true);
 });
+
+//TODO: Delete once drilling on subtotals has been disabled
```

Comment:
Please add JIRA ID to the TODO.

Reply:
We'we agreed to add the removal to BB-1416 acceptance criteria (so we don't mention Jira tasks in this public repo)